README.md

```diff
@@ -1,73 +1,120 @@
-[groups-badge]: https://img.shields.io/badge/Google-Groups-orange.svg
-[groups-url]: https://groups.google.com/forum/#!forum/tarantool
-[telegram-badge]: https://img.shields.io/badge/Telegram-join%20chat-blue.svg
-[telegram-url]: http://telegram.me/tarantool
-[slack-badge]: https://img.shields.io/badge/Slack-join%20chat-lightgrey.svg
-[slack-url]: http://slack.tarantool.org/
-[gitter-badge]: https://badges.gitter.im/Join%20Chat.svg
-[gitter-url]: https://gitter.im/tarantool/tarantool
+<p align="left">
+  <a href="https://tarantool.io/en/">
+    <img
+      alt="Tarantool"
+      src="https://i.imgur.com/BUYqImx.png"
+      width="400"
+    />
+  </a>
+</p>
+
+**A feature-rich in-memory database.**  It has the performance of a key-value
+cache with the functionality you would expect from a regular database:
+
+* SQL
+* ACID
+* primary and secondary indexes
+* disk engine
+* stored procedures
+* replication
+* sharding
+* and a wealth of modules and connectors from popular languages
+
+Another Tarantool feature is the ability to write code in Lua, a general purpose
+programming language, right inside the database. You can write custom expiration
+strategies and any business logic, and synchronize data with other data sources.
+
+You can use Tarantool for any KV purposes -- or use it to replace Redis when
+you need secondary indexes, a different structure of storage data, SQL, etc.
+
+For more details, please see the
+[Tarantool documentation](https://www.tarantool.io/en/doc/2.2/).
+
+## Getting started
+
+### Get hold of Tarantool
+
+* Try online:
+
+  Visit our playground at https://www.tarantool.io/en/try-dev/
+
+* Use a Docker image:
+
+  ```bash
+  docker run -d -p 3301:3301 -v $(pwd):/var/lib/tarantool tarantool/tarantool:2
+  ```
+
+* Install locally:
+
+  * On Ubuntu/Debian/Centos:
+
+    ```bash
+    curl -L http://tarantool.io/installer.sh | VER=2.2 sudo bash
+    ```
+
+  * On Mac OS using [Homebrew](https://brew.sh):
+
+    ```bash
+    brew install tarantool
+    ```
+
+  For more details, please visit https://tarantool.io/download.
+
+* Build from source:
+
+  Please see
+  [detailed instructions](https://tarantool.io/doc/2.2/dev_guide/building_from_source/)
+  in the Tarantool documentation.
+
+### Connectors
+
+We have connectors for many popular languages:
+
+* [Python](https://github.com/tarantool/tarantool-python)
+* [PHP](https://github.com/tarantool-php/client)
+* [Java](http://github.com/tarantool/tarantool-java/)
+* [Go](https://github.com/mialinx/go-tarantool)
+
+[Check out connector details for other languages](https://www.tarantool.io/doc/2.2/book/connectors/)
+and connect to your current instance of Tarantool.
+
+## Use cases
+
+* Transaction cache, OLTP
+* Realtime analytics
+* ETL
+* Profile/session storage
+* Queue
+* IoT
+
+## Ready-made examples
+
+See the [tarantool/examples](https://github.com/tarantool/examples) repository
+with source code of typical projects.
```

Comment:
1. Both examples are on they own branches (planned to be merged?), a user unlikely will find them.
2. Both examples are about using cartridge, not tarantool itself. Let's recommend them from cartridge repository so, it is not much related here.
3. Readme files in Russian (at least [here][1]).

[1]: https://github.com/tarantool/examples/blob/92de6e259888079586f32a59c2e09ee239395199/profile-storage/README.md

Reply:
We can write a brief note about the `Cartridge`.  
Do we have repo with Tarantool examples?

Reply:
Is the section about frameworks (cartridge) or about examples? If the latter, then I would give a link to [pokemon-go manual](https://www.tarantool.io/en/doc/latest/book/app_server/creating_app/).

I don't have objections against mentioning cartridge here.